index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home / Twitter</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <div class="grid-container">
+        <div class="left-sidebar">
+            <div class="leftbar twitterlogo"> <img src="/images/svg/twitter.svg"></div>
+            <div class="leftbar lefticon"><img src="/images/svg/home.svg"><p>Home</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/explore.svg"><p>Explore</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/notifications.svg"><p>Notifications</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/messages.svg"><p>Messages</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/bookmarks.svg"><p>Bookmarks</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/profile.svg"><p>Profile</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/more.svg"><p>More</p></div>
```

Comment:
Perhaps instead of Divs use Buttons, this is for correct semantics also you would be able to focus on the buttons using the Tab key whereas divs don't get focused on